src/main/java/org/geysermc/packconverter/api/utils/ImageUtils.java

```diff
@@ -256,16 +255,17 @@ private static BufferedImage toBufferedImage(Image img) {
      * @return Rotated image
      */
     public static BufferedImage rotate(BufferedImage img, int angle) {
+		final int type = img.getType();
         final double rads = Math.toRadians(-angle);
         final double sin = Math.abs(Math.sin(rads));
         final double cos = Math.abs(Math.cos(rads));
         final int w = (int) Math.floor(img.getWidth() * cos + img.getHeight() * sin);
         final int h = (int) Math.floor(img.getHeight() * cos + img.getWidth() * sin);
-        final BufferedImage rotatedImage = new BufferedImage(w, h, img.getType());
+        final BufferedImage rotatedImage = new BufferedImage(w, h, type == 0 ? 5 : type);
```

Comment:
Why this change?

Reply:
It's a workaround for if something goes wrong.
There's not really a better solution for that afaik.
Problem occurs for example when converting Xmas BDCraft 1.17 64x.